submissions/ASTkach/popup/index.html

```diff
@@ -0,0 +1,157 @@
+                                        >
+                                        <ul class="nav__sub-list">
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="5">
+                                                    <img src="img/icons/calendar.png" alt="" class="nav__img" width="886" height="709" />
+                                                    <p class="nav__text">Calendar</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="6">
+                                                    <img src="img/icons/mail.png" alt="" class="nav__img" width="900" height="539" />
+                                                    <p class="nav__text">Mail</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="7">
+                                                    <img src="img/icons/news.png" alt="" class="nav__img" width="880" height="621" />
+                                                    <p class="nav__text">News</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="8">
+                                                    <img src="img/icons/maps.png" alt="" class="nav__img" width="887" height="710" />
+                                                    <p class="nav__text">Map</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="9">
+                                                    <img src="img/icons/files.png" alt="" class="nav__img" width="886" height="531" />
+                                                    <p class="nav__text">Files</p>
+                                                </a>
+                                            </li>
+                                            <li class="nav__sub-item">
+                                                <a class="nav__sub-link" href="#" target="_blank" rel="nofollow noopener" tabindex="10">
```

Comment:
No need to increment `tabindex` for sibling elements. You can use the same value. Check/play with [this demo](https://codepen.io/a-ostrovnyy/pen/ZExXXvq)